awx/ui_next/src/types.js

```diff
@@ -70,7 +70,7 @@ export const JobTemplate = shape({
   inventory: number,
   job_type: oneOf(['run', 'check']),
   playbook: string,
-  project: number,
+  project: oneOfType([shape({}), number]),
```

Comment:
I'm not sure I understand this.  project can be an object when passed to the form?

Reply:
it was left over from when I went down a rabbit hole and was trying to do some extra refactoring of the form.  I'll save that refactoring for another PR.